src/components/HomePage/BodyContainer.tsx

```diff
@@ -0,0 +1,18 @@
+import * as React from 'react'
+import styled from '@emotion/styled'
+import { neutral } from '@guardian/src-foundations/palette'
+
+import Container from '../Container'
+
+const StyledBodyContainer = styled(Container)`
+  border-top: ${neutral[46]} 0.2px solid;
+  border-left: ${neutral[46]} 0.2px solid;
+  border-right: ${neutral[46]} 0.2px solid;
```

Comment:
Why use a a border-width below `1px`?

Reply:
Good question. These are taken to match the styles used on the header to make sure that it was consistent styling. Where those styles come from is a separate question...